kong/pdk/ctx.lua

```diff
@@ -14,19 +14,19 @@ local _CTX_CORE_KEY = {}
 
 
 ---
--- A table that has the lifetime of the current request and is shared between
+-- A table that contains the history of the current request and is shared between
```

Comment:
I actually think the previous phrasing is more appropriate/accurate here. To explain: the term "lifetime" as used here refers to the programmatic scope of the table's data (it is in a sense returned to a clean slate for each request). It's not meant to imply that the table holds any structured historical data about the request (although that would be valid use-case of the table).